intro.tex

```diff
@@ -1,41 +1,54 @@
+Recent planning on the construction project has led to a reduced amount of on--sky time in commissioning, including a reduction in the time dedicated to final science validation of the as-built system. 
+As Rubin construction moves through the challenging phase of System Integration, Test and Commissioning (SIT-COM), on--sky time could be further reduced. 
+The Operations team is thus planning for various outcomes that might require special attention to producing \es opportunities in the first part of regular operations to ensure the community has prompt access to science-ready data products while the survey begins its relentless coverage of the sky before DR1.
 
+In all cases, it is assumed that the Rubin Construction project delivers an integrated system that can capture, transfer and process science-quality data at the time Operations begins.
+Planning will then consider the following three high-level options to ensure \es:
 \begin{itemize}
-\item {\textit Plan A:} SV is completely successful, move quickly to the LSST and DR1 while providing \es data products in Data Previews
-\item {\textit Plan B:} Early Science Period in the start of full operations (3-6 months) that is different than regular survey operations because on-sky time in SIT-COM is reduced, leading to fewer science ready data before the LSST begins, but the system is otherwise ready
-\item {\textit Plan C:} further shakedown of operations procedures and data taking is required even though the initial condition above is satisfied and the Rubin System can capture and produce science quality data (i.e., the system passes the construction completion requirements but the operations team is not yet ready to begin the LSST).
-
+\item \textbf{Option A}: 
+Science validation is completely successful.
+Begin the LSST working towards DR1 while providing \es data products. 
+\item \textbf{Option B:} 
+On-sky time in commissioning was reduced such that fewer science-ready data products were produced before the LSST begins but the Rubin system can capture and produce science-quality data. 
+Instigate a dedicated Early Science observing campaign for a limited period (3-6 months) at the start of full operations that is different to the regular survey cadence. 
+\item \textbf{Option C:} 
+The Rubin System  passes the construction completion requirements and can capture and produce science-quality data but the operations team is not yet ready to begin the LSST. 
+Further shakedown of operations procedures and data taking is required, which will result in a delay to the start of Operations. 
```

Comment:
delay to the start of full survey (LSST) operations.

comment:(we are in Operations phase).